### ISSUE TITLE
Load material in .obj file

### ISSUE THREAD

**Issue by Flucadetena** [olexale/arkit_flutter_plugin#155]
Hi, I can load the .obj file but can't find a way to load the .mtl file. thanks a lot!

**Comment by olexale** [comment]
Hi! This is probably not the answer you wanted to get but my recommendation is to convert it to `.dae` or `.usdz`. If this is not possible - make sure that the path to the `.mtl` file, which is written in the `.obj` file, is valid and paths to the textures inside the `.mtl` file are valid also.

**Comment by Flucadetena** [comment]
Hi @olexale, coudn't find a way to make it work, but I have a .dae file, so i'll be using that. Thanks anyway for the help and time ;)

**Comment by eduardoGo** [comment]
Hey, the .dae file still need extra files to load the textures... I have the .dae files in the same directory of the textures, but the texture are still not loaded :(.

Context: I'm downloading the .dae files and textures from a remote server.

**Comment by Flucadetena** [comment]
@eduardoGo Hi!, I only tried with the assets loaded in the ´´´ios>Runner>models.scnassets´´´ folder. But check that in the program where the .dae is being generated, the files are imported from the same directory. Some times when creating .obj or .dae the textures are imported from a "textures" folder or something like that, if the path is not exactly the same it won't work.

I don't know if that is the case for you. Hope it helps.

BTW i have to work on this in the coming weeks, so if you find a solution please share. If I find one before I'll share my solution ;)

**Comment by eduardoGo** [comment]
Yep, sure. I created a https://github.com/olexale/arkit_flutter_plugin/issues/171 to track it.

**Comment by ruchitmavani** [comment]
> Hey, the .dae file still need extra files to load the textures... I have the .dae files in the same directory of the textures, but the texture are still not loaded :(.
> 
> Context: I'm downloading the .dae files and textures from a remote server.

can you include code or sample how you are accessing .dae from server? (if possible provide a sample url)
